code/game/objects/items/ego_weapons/teth.dm

```diff
@@ -118,3 +118,18 @@
 /obj/item/ego_weapon/wrist/melee_attack_chain(mob/user, atom/target, params)
 	..()
 	user.changeNext_move(CLICK_CD_MELEE * 0.30)
+
+/obj/item/ego_weapon/regret
+	name = "regret"
+	desc = "Before swinging this weapon, expressing one’s condolences for the demise of the inmate who couldn't even have a funeral would be nice."
+	special = "This weapon attacks extremely slowly."
+	icon_state = "regret"
+	force = 60					//Lots of damage, way less DPS
+	damtype = RED_DAMAGE
+	armortype = RED_DAMAGE
+	attack_verb_continuous = list("beats", "smacks")
+	attack_verb_simple = list("beat", "smack")
+
+/obj/item/ego_weapon/regret/melee_attack_chain(mob/user, atom/target, params)
+	..()
+	user.changeNext_move(CLICK_CD_MELEE * 3.1) // Really Slow. This is the slowest teth we have
```

Comment:
This might be too much.

Reply:
This was me, I designed the ego It's supposed to be a slow one hit weapon that puts you out of combat for like 3 seconds.

Reply:
That's a single WAW level hit at TETH level, trade-off is fair and puts us in about the DPS range/ratio we're looking for

Reply:
I reduced it to 2 since thats +0.4 to red eyes and it still feels unweildy to use. Can always be changed to be longer later on.

Reply:
Actually Higher Damage weapons should have slightly lower DPS. 
This lets you back off and wait for the cooldown to run it's course before running in for massive damage